tools/run-container

```diff
@@ -463,9 +463,13 @@ main() {
         return
     }
 
-    inside_as_cd "$name" root "$cdir" \
-        python3 ./tools/read-dependencies "--distro=${OS_NAME}" \
-            --test-distro || {
+    local rdcmd=(python3 tools/read-dependencies "--distro=${OS_NAME}" --install)
+    if [[ ${OS_NAME} == centos ]]; then
+        error "WARNING: Can't setup CentOS for testing (missing deps, LP: #1867151)."
+    else
+        rdcmd+=('--test-distro')
+    fi
+    inside_as_cd "$name" root "$cdir" "${rdcmd[@]}" || {
```

Comment:
This was object of some discussion. Basically the problem is that some test dependencies are missing from CentOS, and I couldn't find a 3rd party repo with all the needed deps, so we skip them

Even with Py2 we were not running the unittests when building the RPM (fixme?), so this is not a "downgrade", but still not optimal.

Reply:
I think dropping Py2 is good, but here I think we should install every package we know should exist, and only specifically ignore the two that are missing on CentOS.

I agree with the warning message, but we really only need to prevent running unit tests, we should try to install every dependency we can to aid in CentOS dev environment setup as best we can.  Instead of bailing on all test deps, we can specifically prevent running unittests and install all deps that we know should be available.


```suggestion
 local rdcmd=(python3 tools/read-dependencies "--distro=${OS_NAME}" --install --test-distro)
if [[ ${OS_NAME} == centos ]]; then
        error "WARNING: Can't run CentOS unit tests missing pkg deps, LP: #1867151)."
        unittest=""
fi
```